src/foam/dao/Adapter.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+foam.INTERFACE({
+  package: 'foam.dao',
+  name: 'Adapter',
+
+  documentation: `DAO for adapting between "of" input type and "to" delegate
```

Comment:
Documentation looks like for AdapterDAO, not DAOInterface.

Reply:
Updated .. put the correct header too.